sys-apps/tas/tas-1.5.2.180622.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit linux-info systemd
+
+CONFIG_CHECK="~IPMI_DEVICE_INTERFACE ~IPMI_HANDLER ~IPMI_SI"
+
+MY_DATE="$(ver_cut 4)"
```

Comment:
Isn't ver_cut only available in EAPI 7 or eapi7-ver eclass?

Reply:
Cannot you use EAPI 7 btw?

Reply:
No, 'systemd' does not support EAPI=7. At least repoman complains.